src/app/components/Footer/List/index.tsx

```diff
@@ -0,0 +1,44 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { FooterLink } from '#app/models/types/serviceConfig';
+import Link from '../Link';
+import styles, { gridTemplateRows } from './index.styles';
+
+export default ({
+  elements = [],
+  trustProjectLink,
+}: {
+  elements?: (jsx.JSX.Element | null)[];
+  trustProjectLink?: FooterLink;
+}) => {
+  return (
+    <ul
+      role="list"
+      css={[
+        styles.list,
+        trustProjectLink
+          ? [
+              styles.listPaddingWithTrustProjectLink,
+              styles.listItemWithBottomBorder,
+            ]
+          : styles.listPaddingWithoutTrustProjectLink,
+        gridTemplateRows({
+          itemCount: elements.length,
+          trustProjectLink,
+        }),
+      ]}
+    >
+      {trustProjectLink && (
+        <li css={styles.listItem}>
```

Comment:
@greenc05 I have spotted a potential bug with the existing footer (this code is a lift-and-shift from the existing code). You'll notice that this `li` has no role, but below (line 38), it does.

Should all `li`'s have a `role="listitem"`?